clis/teliod/src/core_api.rs

```diff
@@ -71,22 +71,19 @@ fn build_backoff() -> Result<ExponentialBackoff, Error> {
     Ok(ExponentialBackoff::new(backoff_bounds)?)
 }
 
-pub async fn init_with_api(
-    auth_token: &str,
-    interface_name: &str,
-) -> Result<DeviceIdentity, Error> {
+pub async fn init_with_api(auth_token: &str) -> Result<DeviceIdentity, Error> {
     let mut identity_path = dirs::data_local_dir().ok_or(Error::NoDataLocalDir)?;
     identity_path.push("teliod");
     if !identity_path.exists() {
         let _ = create_dir_all(&identity_path);
     }
-    identity_path.push(&format!("{interface_name}.json"));
+    identity_path.push("data.json");
```

Comment:
what's the reason behind this change?

Reply:
Based from the discussions on slack, we want to destroy the old config if we switch users.
Currently, if you change the interface name in the config file, it will create a new identity file leaving the old one behind.